lib/services/auth/tests/modelFilters/visualisation-test.js

```diff
@@ -1,4 +1,6 @@
 import testShareableModel from 'lib/services/auth/tests/utils/testShareableModel';
+import 'lib/models/dashboard';
```

Comment:
What was the reason for this import?

Reply:
Chris said this is needed because the tests don't utilise this model elsewhere anymore due to these changes. Therefore this is needed to register the model with Mongoose.